docs/source/verifiers.rst

```diff
@@ -39,14 +39,67 @@ are given below:
+| Conditions                                        | ChecksumTable | Inline        | Inline + Target |
++---------------------------------------------------+---------------+---------------+-----------------+
+| Data inconsistency due to Ghostferry issuing an   | Yes [3]_      | Yes           | Yes             |
+| incorrect UPDATE on the target database (example: |               |               |                 |
+| encoding-type issues).                            |               |               |                 |
++---------------------------------------------------+---------------+---------------+-----------------+
+| Data inconsistency due to Ghostferry failing to   | Yes           | Yes           | Yes             |
+| INSERT on the target database.                    |               |               |                 |
++---------------------------------------------------+---------------+---------------+-----------------+
+| Data inconsistency due to Ghostferry failing to   | Yes           | Yes           | Yes             |
+| DELETE on the target database.                    |               |               |                 |
++---------------------------------------------------+---------------+---------------+-----------------+
+| Data inconsistency due to rogue application       | Yes           | Sometimes [4]_| Yes             |
+| issuing writes (INSERT/UPDATE/DELETE) against the |               |               |                 |
+| target database.                                  |               |               |                 |
++---------------------------------------------------+---------------+---------------+-----------------+
+| Data inconsistency due to missing binlog events   | Yes           | Sometimes [5]_| Sometimes [5]_  |
+| when Ghostferry is resumed from the wrong         |               |               |                 |
+| binlog coordinates.                               |               |               |                 |
++---------------------------------------------------+---------------+---------------+-----------------+
+| Data inconsistency if Ghostferry's Binlog writing | Yes           | Probably not  | Probably not    |
+| implementation is incorrect and modified the      |               | [6]_          | [6]_            |
+| wrong row on the target (example, an UPDATE is    |               |               |                 |
+| supposed to go to id = 1 but Ghostferry instead   |               |               |                 |
+| issued a query for id = 2). This is an unrealistic|               |               |                 |
+| scenario, but is included for illustrative        |               |               |                 |
+| purposes.                                         |               |               |                 |
++---------------------------------------------------+---------------+---------------+-----------------+
```

Comment:
interesting case to think about 🤔  this is if the binlog writer happens to compose and then write an incorrect row due to an incorrect pagination key?

Reply:
Yes, this would be difficult/impossible to catch with anything other than the ChecksumTable verifier.